api/app/settings/common.py

```diff
@@ -731,7 +731,6 @@
 PREVENT_SIGNUP = env.bool("PREVENT_SIGNUP", default=False)
 PREVENT_EMAIL_PASSWORD = env.bool("PREVENT_EMAIL_PASSWORD", default=False)
 DISABLE_ANALYTICS_FEATURES = env.bool("DISABLE_ANALYTICS_FEATURES", default=False)
-DISABLE_INFLUXDB_FEATURES = env.bool("DISABLE_INFLUXDB_FEATURES", default=True)
```

Comment:
Hmm... removing environment variables like this could be construed as a breaking change. We may have self hosted customers using this environment variable.

We probably have a couple of options here: 

 1. Remove it anyway and make a big point of it in the release notes
 2. Just update the code / setting but still use the same environment variable

I think (2) is probably the way forward but happy to discuss.

Reply:
Agree on the option 2, thanks! Added reading from `DISABLE_INFLUXDB_FEATURES`.